source/shared/net/guides/overview-baker-process.rst

```diff
@@ -1,49 +1,101 @@
 .. _overview-baker:
 
-================================
+=============================
 Overview of the baker process
-================================
+=============================
 
 .. contents::
    :local:
    :backlinks: none
 
-This overview describes the recommended scenario for running a node and becoming a baker on the Concordium blockchain. The guide assumes that you'll be using the Desktop Wallet in combination with a Ledger Nano S hardware wallet to generate baker key and that you'll be running the baker node on a server with Ubuntu.
+Baking is possible with both Mobile Wallet and Desktop Wallet, however the process differs between the two. The overviews below give a brief description of the process.
+
+.. Note::
+
+   If you plan to be a baker, Concordium recommends using the Desktop Wallet since it uses a Ledger hardware device in combination with the wallet to keep your funds secure.
```

Comment:
Is this an official recommendation? We don't want to give the wrong impression that it is not secure to use the other process.

Reply:
Stefan told me that we should recommend Desktop Wallet for baking.

Reply:
Ok. Perhaps I am reading it a bit too harshly. To me it looked like the last part of the sentence is implying that using the mobile wallet will make funds insecure.